app/src/main/java/com/globant/example/mentorapp/presentation/presenter/ListUsersPresenterImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.globant.example.mentorapp.presentation.presenter;
+
+import android.support.annotation.NonNull;
+
+import com.globant.example.mentorapp.data.util.ApiUtils;
+import com.globant.example.mentorapp.domain.interactor.ListUsersInteractorImpl;
+import com.globant.example.mentorapp.presentation.di.Component.UserComponent;
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.view.fragment.ListUsersFragment;
+import com.squareup.otto.Bus;
+import com.squareup.otto.Subscribe;
+
+import javax.inject.Inject;
+
+/**
+ * This class controls the exchange of information between the view and the data sources
+ * Created by oscar.amezquita on 7/06/2017.
+ */
+
+public class ListUsersPresenterImpl implements ListUsersPresenter {
+
+    @Inject
+    ListUsersInteractorImpl interactor;
+
+    private ListUsersFragment listUsersFragment;
+
+    @Inject
+    Bus bus;
+
+    public ListUsersPresenterImpl(UserComponent component) {
+        component.inject(this);
+    }
+
+    @Override
+    public void getUsersFromService(@NonNull ListUsersFragment listUsersFragment, SharedUserViewModel model) {
+        this.listUsersFragment = listUsersFragment;
+        interactor.getUsersListFromService(model);
+
+    }
+
+    @Subscribe
+    public void controlResponse(String result) {
```

Comment:
Using Strings for Events is a bad practice. What if in any other class you're subscribed for String events? which class should be in charge of the event properly?.

Create Event classes to manage specific operations